iron-ajax.html

```diff
@@ -277,8 +277,19 @@
         value = this.params[param];
         param = window.encodeURIComponent(param);
 
-        if (value !== null) {
-          param += '=' + window.encodeURIComponent(value);
+        //if this parameter is an array
+        if( Object.prototype.toString.call( value ) === '[object Array]' ){
+          //we create the queryString differently
+          var URIEncodedArrayItems = [];
+          param = window.encodeURIComponent(param + "[]");
+          for(index in value){
+            URIEncodedArrayItems.push(param + '=' + window.encodeURIComponent(value[index]));
```

Comment:
Is the convention of adding the suffix [] to array param names commonly used anywhere outside of Rails

I'd lean towards saying that users that want that behavior should specify their param name in the params object with the braces (i.e.  `{ "paramName[]": value }`) rather than baking that logic into iron-ajax, as it seems equally reasonable for the server to simply accept multiple params with the same name without the [], which wouldn't otherwise be possible with this PR as is.